usig/sgx/usig-enclave.go

```diff
@@ -16,12 +16,69 @@
 
 package sgx
 
-// #cgo pkg-config: $SGX_URTS_PACKAGE
-// #cgo CFLAGS: -I${SRCDIR}/shim
-// #cgo LDFLAGS: -L${SRCDIR}/shim -lusig_shim
-//
-// #include <stdlib.h>
-// #include "usig.h"
+/*
+// TODO Use pkg-config?
```

Comment:
I'm not sure that pkg-config helps here, but if you choose to drop it, you can also remove lines about pkg-config in `README.md` and `.circleci/config.yml`.

Reply:
To avoid confusion, `pkg-config` would need the exact library name: `libsgx_urts_sim` or `libsgx_urts`, but we just want to get rid of that...

Reply:
I was considering how we add the path of include files of the SDK without hard-coding. Because now we need only the include path (`-I`) (e.g. no `-L` and `-l`), we don't need the code generation like before (i.e. `#cgo pkg-config: libsgx_urts_sim` works for both modes).

Anyway, I will try to remove dependency to the header files in next revision.

Reply:
Removing that dependency would require to avoid using SGX SDK types in `usig.h`. That probably also means translating parts of Go code from `usig-enclave.go` to C code in `shim.c`.